src/foam/u2/view/UnstyledTableView.js

```diff
@@ -257,13 +257,19 @@ foam.CLASS({
                     }
 
                     overlay.forEach(availableActions, function(action) {
-                      this.start()
+                      var elm = this.start()
                         .addClass(view.myClass('context-menu-item'))
-                        .add(action.label)
-                        .on('click', function(evt) {
-                          action.code.call(obj);
-                        })
-                      .end();
+                        .add(action.label);
```

Comment:
If you didn't want to break up the call and store it in the variable 'elm' you could do .call(function() { if ( action.isEnabledFor() { this.on() } else this.addClass('disabled'); })...